docs/send-data/choose-collector-source.md

```diff
@@ -93,6 +93,42 @@ The following table shows the major differences between them.
 |:--|:--|
 | <ul><li>Installed on a system within your deployment locally or remotely.</li><li>Sources collect data available in your deployment.</li><li>Easy to troubleshoot based on Collector logs.</li><li>Supports using Local Configuration File Management so you can use JSON files to configure Sources.</li></ul> | <ul><li>Hosted by Sumo Logic. Agentless: no software to install or activate on a system in your deployment.</li><li>Hosts Sources to collect seamlessly from AWS, Google, and Microsoft products.</li><li>Can receive logs and metrics uploaded via a URL.</li></ul> |
 
+### Compare Installed Collectors and OpenTelemetry Collectors​
+
+The Installed Collector and OpenTelemetry Collector are two popular collectors used for collecting metrics, traces, and logs from various sources. While both collectors have their own unique features and advantages, there are some key differences between them.
+
+**Installed Collector**. The Installed Collector is a standalone agent that runs on Linux, MacOS, Kubernetes, and Windows platforms. It supports a wide range of sources, including Local File, Syslog, Host/Process Metrics, Streaming Metrics, Transaction Tracing, and many more. It also provides support for remote management and configuration, Ingest Budgets, Collector Management API, and CPU targets.
+
+**OpenTelemetry Collector**. The OpenTelemetry Collector is a single-agent management solution that runs on Linux, MacOS, Kubernetes, and Windows platforms. It supports sources such as Local File, Syslog, Host/Process Metrics, Streaming Metrics, and Transaction Tracing. However, it does not provide support for remote management or configuration, Ingest Budgets, Collector Management API, or CPU targets.
+
+#### When to Choose Installed Collector vs. OpenTelemetry Collector
+
+The following table shows the comparison between the Installed Collector and OpenTelemetry Collector based on their supported platforms and sources, and their ideal use cases.
+
+| Collector Type       | Supported Platforms | Supported Sources                                                                                                                       | Ideal Use Cases                                                                                                                                                                          |
+|----------------------|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Please left align

```suggestion
|:-------|:-----------|:------------|:------------------------|
```

Reply:
@kimsauce Table looks okay to me, could you specify?

Reply:
As per my comment, columns must be left-aligned

Reply:
Why can't we keep them center-aligned? Are we following any rules here?

Reply:
Yes, this is our style. Without it, this can cause wonky display issues depending on the desktop/mobile env. I'll add this to the style guide if not there.

Reply:
Oh sorry it already was in style guide https://help.sumologic.com/docs/contributing/style-guide/#tables